runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/HotKeyLogger.java

```diff
@@ -52,7 +52,9 @@ public void logHotKeyDetection(String userStepName, Duration hotKeyAge) {
     LOG.warn(
         "A hot key was detected in step '{}' with age of '{}'. This is "
             + "a symptom of key distribution being skewed. To fix, please inspect your data and "
-            + "pipeline to ensure that elements are evenly distributed across your key space.",
+            + "pipeline to ensure that elements are evenly distributed across your key space. If "
+            + "you want to log the plain-text key to Cloud Logging please re-run with the "
+            + "`hotKeyLoggingEnabled` pipeline option.",
```

Comment:
LGTM. Do we also want to refer the user to the set method? https://github.com/apache/beam/blob/master/runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/options/DataflowPipelineOptions.java#L207

Reply:
Sure, I added a link to the Cloud docs about specifying pipeline options. This generalizes across different languages and is a good how-to guide.